stdlib/int.mli

```diff
@@ -0,0 +1,140 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Integer values.
+
+    Integers are {!Sys.int_size} bits wide and use two's complement
+    representation. All operations are taken modulo
+    2{^[Sys.int_size]}. They do not fail on overflow.
+
+    @since 4.08 *)
+
+(** {1:ints Integers} *)
+
+type t = int
+(** The type for integer values. *)
+
+val zero : int
+(** [zero] is the integer [0]. *)
+
+val one : int
+(** [one] is the integer [1]. *)
+
+val minus_one : int
+(** [minus_one] is the integer [-1]. *)
+
+external neg : int -> int = "%negint"
```

Comment:
Do we still expose `external` declarations in `.mli` files? Isn't that practice deprecated? IIRC it can cause problems when linking libraries (modules get omitted even though something depends on them) and the performance gain is null in native code. cc @garrigue @xavierleroy @chambart

Reply:
> (modules get omitted even though something depends on them)

I think this was fixed by #289.

Reply:
@damiendoligez This was recently discussed in [this PR](https://github.com/ocaml/ocaml/pull/2005).

Reply:
OK, thanks to you both.